docs/source/schema-design/federated-schemas/reference/directives.mdx

```diff
@@ -1164,6 +1164,82 @@ The default value is `true`.
 </tbody>
 </table>
 
+## Managing cache invalidation
+
+### `@cacheTag`
+
+<MinVersionBadge version="Federation v2.12" />
+
+```graphql
+directive @cacheTag(format: String!) repeatable on FIELD_DEFINITION | OBJECT
+```
+
+Assigns cache tags to cached data in the Apollo Router for [active cache invalidation](/graphos/routing/performance/caching/response-caching/invalidation#active-invalidation). Use cache tags to remove specific cached entries on demand when data changes, rather than waiting for time-to-live (TTL) expiration.
+
+```graphql {6-7, 10}
+extend schema
+  @link(url: "https://specs.apollo.dev/federation/v2.12",
+        import: ["@key", "@cacheTag"])
+
+type Query {
+  users: [User!]! @cacheTag(format: "users-list")
+  user(id: ID!): User @cacheTag(format: "user-{$args.id}")
+}
+
+type User @key(fields: "id") @cacheTag(format: "user-{$key.id}") {
+  id: ID!
+  name: String!
+}
+```
+
+<Note>
+
+The `@cacheTag` directive is defined in your subgraph schema, but it's only interpreted by the router. Your subgraph doesn't use it at all. This differs from `@cacheControl`, which is both defined and interpreted at the subgraph level.
+
+</Note>
+
+#### Arguments
+
+<table class="field-table">
+  <thead>
+    <tr>
+      <th>Name /<br/>Type</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+
+<tr class="required">
+<td>
+
+##### `format`
+
+`String!`
+
+</td>
+<td>
+
+**Required.** A string template that defines the cache tag. Can include interpolated variables:
+- For root fields: `{$args.fieldName}` to interpolate field arguments
+- For entities: `{$key.fieldName}` to interpolate entity key fields
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+<Note>
+
+Enum types are supported in the `format` argument. If you encounter composition errors when using enums, use Apollo Router v2.8.2 or later, which includes a fix for enum validation.
+
+</Note>
+
+**Programmatic tags:** To set cache tags programmatically (when tags don't depend on root field arguments or entity keys), add `apolloEntityCacheTags` or `apolloCacheTags` to your subgraph response extensions. For details, see [Programmatic cache tags](/graphos/routing/performance/caching/response-caching/invalidation#by-cache-tag).
+
+For complete information on cache invalidation, including how to send invalidation requests to the router, see [Response cache invalidation](/graphos/routing/performance/caching/response-caching/invalidation).
```

Comment:
```suggestion
For more information on how to apply cache invalidation with cache tags, see the documentation on [response cache invalidation](/graphos/routing/performance/caching/response-caching/invalidation#by-cache-tag).
```
Again don't want to repeat information already in the other doc so we can bundle both paragraphs into one here.